diff --git a/bindings/python/CMakeLists.txt b/bindings/python/CMakeLists.txt
--- a/bindings/python/CMakeLists.txt
+++ b/bindings/python/CMakeLists.txt
@@ -109,8 +109,7 @@ if (python-egg-info)
  configure_file(${SETUP_PY_IN} ${SETUP_PY} @ONLY)
 
  add_custom_command(OUTPUT ${OUTPUT}
- COMMAND ${Python3_EXECUTABLE} ${SETUP_PY} build -b "${CMAKE_CURRENT_SOURCE_DIR}"
- COMMAND ${Python3_EXECUTABLE} ${SETUP_PY} egg_info -b "${CMAKE_CURRENT_SOURCE_DIR}"
+ COMMAND ${Python3_EXECUTABLE} ${SETUP_PY} egg_info
  COMMAND ${CMAKE_COMMAND} -E touch ${OUTPUT}
  DEPENDS ${DEPS}
  )
